apps/marketing/src/components/icons/drones/micro-decoy-swarm.md

```
@@ -0,0 +1,42 @@
+# Micro-Decoy Swarm Icon
+
+## Description
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)